test/test.js

```diff
@@ -15,3 +17,27 @@ test('Basic test', (t) => {
     t.end()
   })
 })
+
+test('Cache directory is moved to new location', (t) => {
+  // Download to old cache directory location
```

Comment:
I don't know whether this is the right way to write this test. Perhaps if the user specifies a custom cache directory, the migration code shouldn't be triggered.

Regardless of the decision reached for the above question, I think a better test would be:

1. Create `$HOME/.electron` if it doesn't exist, and put a temporary file in that folder
2. Invoke `download()`
3. Assert that the temporary file exists in the new default cache directory
4. Delete temporary file